src/GraphQL/DataObjectQueryFieldConfigGenerator/Datetime.php

```diff
@@ -15,6 +15,37 @@
 
 namespace Pimcore\Bundle\DataHubBundle\GraphQL\DataObjectQueryFieldConfigGenerator;
 
-class Datetime extends Base
+use GraphQL\Type\Definition\ResolveInfo;
+use Pimcore\Bundle\DataHubBundle\GraphQL\Exception\InvalidFieldDefinitionException;
+use Pimcore\Bundle\DataHubBundle\GraphQL\Service;
+use Pimcore\Model\DataObject\ClassDefinition;
+use Pimcore\Model\DataObject\ClassDefinition\Data;
+
+final class Datetime extends Base
 {
+    /**
+     * @param string $attribute
+     * @param ClassDefinition|null $class
+     * @param object|null $container
+     *
+     * @return array
+     *
+     * @throws InvalidFieldDefinitionException
+     */
+    public function getGraphQlFieldConfig($attribute, Data $fieldDefinition, $class = null, $container = null)
+    {
+        if (!($fieldDefinition instanceof Data\Datetime)) {
```

Comment:
The extra parenthesis are not needed but I like it in terms of clarity

Reply:
Agreed, better to read imo